doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -304,8 +304,8 @@ Hints used by :tacn:`auto`, :tacn:`eauto` and other tactics are stored in hint
 databases.  Each database maps head symbols to a list of hints.  Use the
 :cmd:`Print Hint` command to view a database.
 
-Each hint has a cost that is a nonnegative
-integer and an optional pattern. Hints with lower costs are tried first.
+Each hint has a cost and an optional pattern. Hints with lower
+cost are tried first.  (Cost is not used to limit the scope of searches.)
```

Comment:
What does that added "Cost is not used to limit the scope of searches." means?

Reply:
To me, "cost" might lead you to expect that the search process adds the cost of each successful tactic and quits searching when the total exceeds some maximum amount, which is not the case.

Reply:
BTW I just tweaked the changelog to be more accurate.  Good to go.

Reply:
Ok, got it. I didn't got it at first reading but I don't have anything better to offer.